src/clj/rems/api/organizations.clj

```diff
@@ -1,22 +1,28 @@
 (ns rems.api.organizations
   (:require [compojure.api.sweet :refer :all]
-            [rems.api.schema :refer [OrganizationArchivedCommand OrganizationEnabledCommand OrganizationFull SuccessResponse UserId UserWithAttributes]]
+            [rems.api.schema :refer [OrganizationArchivedCommand OrganizationEnabledCommand OrganizationFull SuccessResponse User UserWithAttributes]]
             [rems.api.util] ; required for route :roles
             [rems.api.services.organizations :as organizations]
             [rems.util :refer [getx-user-id]]
             [ring.util.http-response :refer :all]
             [schema.core :as s]))
 
 (s/defschema CreateOrganizationCommand
-  OrganizationFull)
+  (-> OrganizationFull
+      (dissoc :organization/modifier
+              :organization/last-modifier)
+      (assoc (s/optional-key :organization/owners) [User])))
 
 (s/defschema CreateOrganizationResponse
   {:success s/Bool
    (s/optional-key :organization/id) s/Str
    (s/optional-key :errors) [s/Any]})
 
 (s/defschema EditOrganizationCommand
-  OrganizationFull)
+  (-> OrganizationFull
+      (dissoc :organization/modifier
+              :organization/last-modifier)
+      (assoc (s/optional-key :organization/owners) [User])))
```

Comment:
so this is the same as CreateOrganizationCommand?

Reply:
Yeah since it is easy to copy that schema to the name. I'll do that. They ended up the same.